superset/templates/appbuilder/navbar_right.html

```diff
@@ -87,3 +87,10 @@
     <li><a href="{{appbuilder.get_url_for_login}}">
     <i class="fa fa-fw fa-sign-in"></i>{{_("Login")}}</a></li>
 {% endif %}
+
+<li> 
+  <a 
+  href="/static/help/index.html">
```

Comment:
If there is a change corresponding to the above change in path, please modify the path here as well